code/game/objects/items/chrono_eraser.dm

```diff
@@ -243,30 +246,30 @@
 	else
 		return 0
 
-/obj/effect/chrono_field/assume_air()
+/obj/structure/chrono_field/assume_air()
 	return 0
 
-/obj/effect/chrono_field/return_air() //we always have nominal air and temperature
+/obj/structure/chrono_field/return_air() //we always have nominal air and temperature
 	var/datum/gas_mixture/GM = new
 	GM.add_gases(/datum/gas/oxygen, /datum/gas/nitrogen)
 	GM.gases[/datum/gas/oxygen][MOLES] = MOLES_O2STANDARD
 	GM.gases[/datum/gas/nitrogen][MOLES] = MOLES_N2STANDARD
 	GM.temperature = T20C
 	return GM
 
-/obj/effect/chrono_field/Move()
```

Comment:
Why does this need to override move, sounds like absolute overkill.

Reply:
Good point, I'll re*move* this override.